erpnext/patches/v13_0/custom_fields_for_taxjar_integration.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import unicode_literals
+import frappe
+from frappe.custom.doctype.custom_field.custom_field import create_custom_fields
+from erpnext.regional.united_states.setup import add_permissions
+
+def execute():
+	company = frappe.get_all('Company', filters = {'country': 'United States'}, fields=['name'])
```

Comment:
@deepeshgarg007 @nemesis189 

This patch should also be checking that such integration is even enabled right?

Reply:
I assumed the product tax codes would be standard tax codes across the country similar to HSN we have in India. After some research I realize thats not the case. These are only specific to TaxJar. Will make the changes to execute this only if Tax Jar Integration is enabled